tests/integration/defs/triton_server/test_triton_llm.py

```
@@ -2223,738 +2029,12 @@ def test_llama_v3_speculative_decoding_bls(
-        PREPROCESSING_INSTANCE_COUNT,
-        POSTPROCESSING_INSTANCE_COUNT,
-        ACCUMULATE_TOKEN,
-        BLS_INSTANCE_COUNT,
-        ENABLE_CONTEXT_FMHA_FP32_ACC=ENABLE_CONTEXT_FMHA_FP32_ACC,
-    )
-    # WAR for https://nvbugspro.nvidia.com/bug/4742149
-    gpu_name = query_gpu_name()
-    if "NVIDIA H20" == gpu_name:
-        check_call("pip3 install -U nvidia-cublas-cu12", shell=True)
+    # WAR for https://nvbugspro.nvidia.com/bug/4742149
+    gpu_name = query_gpu_name()
```

Flake8 (7.3.0)

[error] 'query_gpu_name' may be undefined, or defined from star imports: .build_engines, .common

(F405)